recipe.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+    <head></head>
+    <body>
+        <h1>The Perfect PB&J</h1>
+        <hr>
+        <img src = "https://i1.wp.com/theheritagecook.com/wp-content/uploads/2013/10/PBJ-Triangles-w-fruit-iStock.jpg?w=637&ssl=1" height = "200" widith = "200"/>
```

Comment:
get into the habit of adding `alt="any description of the image"` to you img tags, this will throw errors later on.